### DISCUSSION TITLE
Is It Possible to Remove Discussion?

### DISCUSSION THREAD

**Discussion by eren** [giscus/giscus#1317] [Q&A] [answered]
While testing the app on local instance, I was able to send a comment to GitHub but now it is permanent. I can edit the post but I cannot delete the original post made by the bot.

Is it possible to remove this item so that I can start a clean slate after everything is set up?

Thank you,
Eren

**Comment by razonyang** [comment, marked as answer]
You can delete the discussion via _Delete discussion_ button on the right sidebar.

> If there is no such button, please make sure if you have the right permission on the corresponding repository.

![image](https://github.com/giscus/giscus/assets/17720932/767cee51-a296-4a36-807a-9fa76131dd28)
